src/lotus_json/actor_states/methods/miner_actor_params.rs

```diff
@@ -0,0 +1,3748 @@
+    pub new_quota: TokenAmount,
+    pub new_expiration: ChainEpoch,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ChangeOwnerAddressParamsLotusJson(
+    #[schemars(with = "LotusJson<Address>")]
+    #[serde(with = "crate::lotus_json")]
+    Address,
+);
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct CompactPartitionsParamsLotusJson {
+    pub deadline: u64,
+    #[schemars(with = "LotusJson<BitField>")]
+    #[serde(with = "crate::lotus_json")]
+    pub partitions: BitField,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct CompactSectorNumbersParamsLotusJson {
+    #[schemars(with = "LotusJson<BitField>")]
+    #[serde(with = "crate::lotus_json")]
+    pub mask_sector_numbers: BitField,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct DisputeWindowedPoStParamsLotusJson {
+    pub deadline: u64,
+    #[serde(rename = "PoStIndex")]
+    pub post_index: u64,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExtendSectorExpirationParamsV8LotusJson {
+    pub extensions: Vec<ExpirationExtensionV8LotusJson>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExtendSectorExpirationParamsLotusJson {
+    pub extensions: Vec<ExpirationExtensionLotusJson>,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExpirationExtensionV8LotusJson {
+    pub deadline: u64,
+    pub partition: u64,
+    #[schemars(with = "LotusJson<Vec<u8>>")]
+    #[serde(with = "crate::lotus_json")]
+    pub sectors: Vec<u8>,
+    pub new_expiration: ChainEpoch,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExpirationExtensionLotusJson {
+    pub deadline: u64,
+    pub partition: u64,
+    #[schemars(with = "LotusJson<BitField>")]
+    #[serde(with = "crate::lotus_json")]
+    pub sectors: BitField,
+    pub new_expiration: ChainEpoch,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExtendSectorExpiration2ParamsLotusJson {
+    pub extensions: Vec<ExpirationExtension2LotusJson>,
+}
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ExpirationExtension2LotusJson {
+    pub deadline: u64,
+    pub partition: u64,
+    #[schemars(with = "LotusJson<BitField>")]
+    #[serde(with = "crate::lotus_json")]
+    pub sectors: BitField,
+    // Lotus returns null (not []) when there are no sector claims; model as Option, so None means empty.
```

Comment:
(Nothing to change) 
Attributing `#[serde(with = "crate::lotus_json")]` to `Vec` should achive the same